We're only looking at the top-level `features` directory by default, we should recurse through child directories

We should recurse through the default `./features` directory by default, we should make the code search subdirectories for .feature files
```
.paniconexit0 -test.run ^\QTestCheckout\E$
=== RUN   TestCheckout
    runner_test.go:47: no tests found in paths: []
--- FAIL: TestCheckout (0.00s)
```

--- a/README.md
+++ b/README.md
@@ -185,7 +185,7 @@ tests. `Runner.ShortTags` method can be used to select a set of tags to
 
 `Runner` has the following methods for setting custom options
 
-* `Path()` sets custom paths. The default is `features/*.feature`.
+* `Path()` sets custom paths, and accepts `**` glob patterns making use of the [doublestar](https://github.com/bmatcuk/doublestar/tree/master) library. The default is `features/**/*.feature`
 * `Step()` can be used to add custom steps with special regular expressions.
 * `Before()`, `After()`, `BeforeStep()`, or and `AfterStep()` can be used to register custom hooks.
 * `Tags` and `ShortTags` can be used with tag expressions as described above.

--- a/_examples/subdirectories/features/one/one.feature
+++ b/_examples/subdirectories/features/one/one.feature
@@ -1,0 +1,6 @@
+Feature: one
+
+  Scenario: eat cukes
+    Given I have 10 cukes
+    When I eat 1
+    Then I have 9 left

--- a/_examples/subdirectories/features/two/two.feature
+++ b/_examples/subdirectories/features/two/two.feature
@@ -1,0 +1,6 @@
+Feature: two
+
+  Scenario: eat cukes
+    Given I have 10 cukes
+    When I eat 2
+    Then I have 8 left

--- a/_examples/subdirectories/features/zero.feature
+++ b/_examples/subdirectories/features/zero.feature
@@ -1,0 +1,6 @@
+Feature: zero
+
+  Scenario: eat cukes
+    Given I have 10 cukes
+    When I eat 0
+    Then I have 10 left

--- a/_examples/subdirectories/runner_test.go
+++ b/_examples/subdirectories/runner_test.go
@@ -1,0 +1,31 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/regen-network/gocuke"
+)
+
+// should recursively load scenarios from features directory
+func TestSimple(t *testing.T) {
+	gocuke.NewRunner(t, &simpleSuite{}).Run()
+}
+
+type simpleSuite struct {
+	gocuke.TestingT
+	cukes int64
+}
+
+func (s *simpleSuite) IHaveCukes(a int64) {
+	s.cukes = a
+}
+
+func (s *simpleSuite) IEat(a int64) {
+	s.cukes -= a
+}
+
+func (s *simpleSuite) IHaveLeft(a int64) {
+	if a != s.cukes {
+		s.Fatalf("expected %d cukes, have %d", a, s.cukes)
+	}
+}

--- a/go.mod
+++ b/go.mod
@@ -3,6 +3,7 @@ module github.com/regen-network/gocuke
 go 1.21
 
 require (
+	github.com/bmatcuk/doublestar/v4 v4.6.1
 	github.com/cockroachdb/apd/v3 v3.2.1
 	github.com/cucumber/common/messages/go/v19 v19.1.2
 	github.com/cucumber/gherkin/go/v26 v26.2.0

--- a/run.go
+++ b/run.go
@@ -2,11 +2,11 @@ package gocuke
 
 import (
 	"os"
-	"path/filepath"
 	"reflect"
 	"strings"
 	"testing"
 
+	"github.com/bmatcuk/doublestar/v4"
 	gherkin "github.com/cucumber/gherkin/go/v26"
 	"gotest.tools/v3/assert"
 )
@@ -17,7 +17,7 @@ func (r *Runner) Run() {
 
 	paths := r.paths
 	if len(paths) == 0 {
-		paths = []string{"features/*.feature"}
+		paths = []string{"features/**/*.feature"}
 	}
 
 	haveTests := false
@@ -29,7 +29,7 @@ func (r *Runner) Run() {
 		// not doing this allows mis-spellings in exact paths to be skipped silently
 		if strings.Contains(path, "*") {
 			var err error
-			files, err = filepath.Glob(path)
+			files, err = doublestar.FilepathGlob(path)
 			assert.NilError(r.topLevelT, err)
 		} else {
 			files = []string{path}

